data-gen/sdc/csvReviews-sdc.js

```diff
@@ -32,13 +32,13 @@ let writeNreviews = (writer, encoding, callback) => {
       let num_comments = generateRandomNum(0, 300, false);
       let num_found_funny = generateRandomNum(0, 200, false);
       let num_found_helpful = generateRandomNum(0, 100, false);
-      let hours_at_review_time = generateRandomNum(100, 1000, true);
-      let hours_on_record = generateRandomNum(100, 1000, true);
+      let hours_at_review_time = generateRandomNum(1000, 5000, true, 100);
```

Comment:
Another option would be to add the division by 100 here, if it's only applicable to this generator.

Reply:
That is viable, to justify making it into a parameter, I was looking in the context of, "This helper function should be reuseable no matter if its for a whole number or float number use case" but your point is totally valid